internal/junitxml/report.go

```diff
@@ -33,6 +33,7 @@ type JUnitTestSuite struct {
 	XMLName    xml.Name        `xml:"testsuite"`
 	Tests      int             `xml:"tests,attr"`
 	Failures   int             `xml:"failures,attr"`
+	Skipped    int             `xml:"skipped,attr"`
```

Comment:
```suggestion
	Skipped    int             `xml:"skipped,attr,omitempty"`
```


I would prefer to do not have the attribute when not needed

Reply:
Done!

Reply:
Thanks